packages/schematics/angular/application/files/src/main.ts.template

```diff
@@ -7,6 +7,12 @@ import { environment } from './environments/environment';
 if (environment.production) {
   enableProdMode();
 }
-
+<% if(!!viewEncapsulation) { %>
+platformBrowserDynamic().bootstrapModule(AppModule, {
+  defaultEncapsulation: ViewEncapsulation.ShadowDom
```

Comment:
Shouldn't this be the view encapsulation value?

Reply:
done